apsuite/commisslib/meas_touschek_lifetime.py

```diff
@@ -718,9 +732,13 @@ def _do_measure(self):
             # Get other relevant parameters
             meas['current'].append(curr.current)
             meas['rf_voltage'].append(rfcav.dev_cavmon.gap_voltage)
+            meas['rf_frequency'].append(rfgen.frequency)
+            meas['sync_tune'].append(bbbl.sram.spec_marker1_tune)
+            meas['sync_frequency'].append(bbbl.sram.spec_marker1_freq)
+
             meas['avg_pressure'].append(press.value)
 
-            if not idx % 100 and parms.save_partial:
+            if not idx % int(parms.save_each_nrmeas) and parms.save_partial:
                 # 5 iterations of orbit correction.
```

Comment:
theses infos seem to be useful for the user. Maybe we could think of an interface where to put this kind of information about the class usage? Maybe this is something we could do for all classes. What do you think about putting it in the class docstring, @murilobalves and @xresende ?

Reply:
yes, I think it is a good idea to have these measurement requirements explicitly explained in the class docstring.

Reply:
Nice suggestion! I updated the class docstring, please take a look. @fernandohds564 @xresende 

I recently found out a cool tool to access classes info directly from terminal. For example, is the branch of this PR:

`from apsuite.commisslib.meas_touschek_lifetime import MeasTouschekLifetime
help(MeasTouschekLifetime)`

prints the class docstring, the class methods with corresponding docstring, the inherited methods from the classes which the present class is derived and so on...